install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -467,6 +467,10 @@ metadata:
 spec:
   host: istio-policy.{{ .Release.Namespace }}.svc.cluster.local
   trafficPolicy:
+    {{- if .Values.global.controlPlaneSecurityEnabled }}
```

Comment:
Given that we have tls and non tls port for mixer, don’t you want to use port level setting here ?

Reply:
+ @mandarjog @kyessenov Do you know what ports should be in mTLS or not?

Reply:
9091 is always non-mTLS, and 15004 is either depending on control plane security option.

Don't you need the same thing for pilot?

Reply:
Please don't hardcode the value though. The ports are called `grpc-mixer` and `grpc-mixer-mtls`.

Reply:
Done.